### ISSUE TITLE
Vertex ordering in function make_icosahedron() is wrong

### ISSUE THREAD

**Issue by unclejimbo** [CGAL/cgal#2975]
## Issue Details

The vertex ordering is cw instead of ccw such that all face normals are inverted, see picture below:
![](http://7xllm5.com1.z0.glb.clouddn.com/icosahedron.png)
I inverted the ordering of one face and its normal is correct.

## Source Code

I output the mesh into an off file as follows

```
using Kernel = CGAL::Simple_cartesian<float>;
using Point_3 = Kernel::Point_3;
using Mesh = CGAL::Surface_mesh<Point_3>;
Mesh mesh;
CGAL::make_icosahedron<Mesh>(mesh, Point_3(0, 0, 0));
std::ofstream ofs("path/to/mesh.off");
ofs << mesh;
```

## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits):
* Compiler:
* Release or debug mode:
* Specific flags used (if any):
* CGAL version: 4.11.1
* Boost version:
* Other libraries versions if used (Eigen, TBB, etc.):

**Comment by lrineau** [comment]
@maxGimeno I think you are the author. Could you try to reproduce (or not), and eventually fix. The bug is against CGAL-4.11.1. Please work the `releases/CGAL-4.11-branch`.

**Comment by afabri** [comment]
It is not an error, as it is not specified, but  I agree that all the funcions that generate a mesh should have the same orientation, and that must be documented.

**Comment by lrineau** [comment]
Fixed by #2976.
